src/ir/transform.cc

```diff
@@ -531,6 +531,23 @@ Pass CreateModulePass(const runtime::TypedPackedFunc<IRModule(IRModule, PassCont
   return ModulePass(pass_func, pass_info);
 }
 
+Pass ApplyPassToFunction(Pass pass, String func_name, bool error_if_function_missing) {
```

Comment:
This function seems useful. Can we get the `List[String]` instead of single function name so that we can generalize a little further?

Reply:
I think that would work, but only in the case where `error_if_function_missing = false`.  If there's a `List[String]`, then it wouldn't be clear to a reader whether an error is raised when all of the listed functions are absent, or when any of the listed functions are absent.

I've been picturing this utility as being useful for hand-crafted optimization pipelines, where some optimizations should only be applied to specific functions.  In those cases, raising an error when an expected function is missing would allow for earlier detection of an invalid optimization pipeline.

Reply:
As a third option, what if the `func_name` parameter is replaced with a `func_name_regex` parameter?  That way, it could handle the cases of a single function and of a list of functions, and could clearly state when an error occurs by renaming the parameter to `error_if_no_function_matches_regex`.

Reply:
And the more I think on it, the more I like the regex option.  Implemented, and going through CI now.